src/bin/vip-dev-env-update.js

```diff
@@ -76,7 +76,12 @@ cmd.argv( process.argv, async ( arg, opt ) => {
 			title: '',
 		};
 
-		const instanceData = await promptForArguments( preselectedOptions, defaultOptions );
+		const providedOptions = Object.keys( opt )
+			.filter( option => option.length > 1 ) // Filter out single letter aliases
```

Comment:
Why are we filtering out the single-letter aliases?

Reply:
The library we use to parse arguments always populates both no matter if you actually used single letter or full world one.

It doesn't matter too much but I would had to add also the one letter version variants to the arguments we should ignore without it.